torch/nn/_functions/conv.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, stride, padding, dilation, transposed, output_padding,
         self.groups = groups
 
     def forward(self, input, weight, bias=None):
+        if not input.is_contiguous():
+            raise ValueError("Convolution input has to be contiguous")
```

Comment:
until the cudnn bindings are changed to use separate descriptors for input and gradOutput, this is presumably needed.